arch/xtensa/src/esp32/Kconfig

```
@@ -531,4 +533,24 @@ config ESP32_ETH_PHY_ADDR
 
 endmenu # ESP32_EMAC
 
+menu "WiFi configuration"
+	depends on ESP32_WIRELESS
+
+config ESP32_WIFI_SAVE_PARAM
+	bool "Save WiFi Parameters"
+	default n
+	help
+		Enable this option, WiFi adapter save WiFi parameters
+		into file system without generating or calculating some
+		connection parameters again.
```

```suggestion
		If you enable this option, WiFi adapter parameters will be saved
		into the file system instead of computing them each time.
```

Changed a bit the wording. Maybe you could specify which parameters?

---

Great, I will list the more important parameters for users, but some background parameters for wifi, I think they are not needed to be mentioned, because users don't use this directly and they event don't care about this.